frontend-react/src/pages/submissions/SubmissionDetails.test.tsx

```diff
@@ -0,0 +1,115 @@
+import { MatcherFunction, screen } from "@testing-library/react";
+import ActionDetailsResource from "../../resources/ActionDetailsResource";
+import { success } from "../../resources/fixtures/ActionDetailFixtures";
```

Comment:
agree with sonarcloud and github: can we remove the unused import?

Reply:
Handled :)